ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/state/beaconstate/versions/capella/BeaconStateSchemaCapella.java

```diff
@@ -43,14 +45,19 @@
   }
 
   private static List<SszField> getUniqueFields(final SpecConfig specConfig) {
+    final SszField latestExecutionPayloadHeaderField =
+        new SszField(
+            LATEST_EXECUTION_PAYLOAD_HEADER_FIELD_INDEX,
+            BeaconStateFields.LATEST_EXECUTION_PAYLOAD_HEADER,
+            () -> new ExecutionPayloadHeaderSchemaCapella(SpecConfigCapella.required(specConfig)));
     final SszField latestWithdrawalValidatorIndexField =
         new SszField(
             LATEST_WITHDRAWAL_VALIDATOR_INDEX,
             BeaconStateFields.LATEST_WITHDRAWAL_VALIDATOR_INDEX,
             () -> SszPrimitiveSchemas.UINT64_SCHEMA);
     return Stream.concat(
-            BeaconStateSchemaBellatrix.getUniqueFields(specConfig).stream(),
-            Stream.of(latestWithdrawalValidatorIndexField))
+            BeaconStateSchemaAltair.getUniqueFields(specConfig).stream(),
```

Comment:
shouldn't be `BeaconStateSchemaBellatrix.getUniqueFields` ?

Reply:
I see the problem here, an option could be to rebuild the list and overwrite BeaconStateFields.LATEST_EXECUTION_PAYLOAD_HEADER with the capella one and then add the 
 LATEST_WITHDRAWAL_VALIDATOR_INDEX. But it would be just for the sake of always building on top of the previous fork, and could worse to read.